x-pack/test/cases_api_integration/common/lib/api/files.ts

```diff
@@ -84,14 +96,66 @@ export const deleteAllFiles = async ({
   expectedHttpCode?: number;
   auth?: { user: User; space: string | null };
 }) => {
-  const files = await listFiles({ supertest, params: { kind }, auth, expectedHttpCode });
+  const { body: files } = await supertest
+    .post(`${getSpaceUrlPrefix(auth.space)}${fileApiRoutes.getFindRoute()}`)
+    .set('kbn-xsrf', 'true')
+    .query({ perPage: 10000 })
+    .auth(auth.user.username, auth.user.password)
+    .send({
+      kind,
+    })
+    .expect(expectedHttpCode);
 
-  await deleteFiles({
-    supertest,
-    files: files.files.map((fileInfo) => ({ kind, id: fileInfo.id })),
-    auth,
-    expectedHttpCode,
-  });
+  const castedFiles = files as Awaited<ReturnType<BaseFilesClient['find']>>;
+
+  if (castedFiles.files.length > 0) {
+    await deleteFiles({
+      supertest,
+      files: castedFiles.files.map((fileInfo) => fileInfo.id),
+      auth,
+      expectedHttpCode,
+    });
+  }
+};
+
+export const deleteAllFiles = async ({
```

Comment:
This is mostly used for clean up to remove the files once the tests are completed.